cmd/erasure-healing.go

```diff
@@ -438,7 +435,12 @@ func (er erasureObjects) healObject(ctx context.Context, bucket string, object s
 			continue
 		}
 		copyPartsMetadata[i] = partsMetadata[i]
-		partsMetadata[i] = cleanFileInfo(latestMeta)
+		if !latestMeta.IsRemote() {
```

Comment:
We can make this block outside for block. It will make it little easier to read

Reply:
we do not need this, this is actually incorrect. cleanFileInfo is correct just that a comment is needed to indicate why that is needed.